styles/vikunja/catppuccin.user.css

```diff
@@ -0,0 +1,282 @@
+/* ==UserStyle==
+@name Vikunja Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/vikunja
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/vikunja
+@version 0.0.1
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/vikunja/catppuccin.user.css
+@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Avikunja
+@description Soothing pastel theme for Vikunja
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire*", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+@-moz-document domain('vikunja.guaxinim.online') {
```

Comment:
I think its best if we add multiple domains, like your own and `try.vukunja.io` and others if people are willing to add them similar to how searxng operates.

Reply:
Yup. I forgot to change from the url of my instance.